packages/application-shell/src/components/navbar/navbar.js

```diff
@@ -325,9 +320,7 @@ export class DataMenu extends React.PureComponent {
             ),
             uriPath: PropTypes.string.isRequired,
             featureToggle: PropTypes.string,
-            permissions: PropTypes.arrayOf(
-              PropTypes.oneOf(Object.keys(permissions))
-            ),
```

Comment:
This is the same prop-type now elsewhere until it his `RestrictedByPermission` which then gives a prop-type warning given the permission is not configured with a new component tree in the stack.